squirrel/commands/watch.py

```diff
@@ -82,11 +83,14 @@ def file_not_exists(files, logger):
 
 def daemon(wd, logger):
     watches = wd
+    # Loads '.ignore' into a variable
+    ignores = Plugin.import_ignores(wd, '.ignore', logger)
```

Comment:
It's better the define the name of the ignore file in `vars.py`, so that if we need to modify it we don't have to hunt for every `'.ignore'` in the code base.

Reply:
Should be resolved in d9d2ca6 - Add 'ignore' in 'vars.py' and updated 'watch.py'